swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category = "Clothing", condition=0):
```

Comment:
Hmm, why aren't we using inheritance for these child classes? The parent class `Item` already has the ability to construct the attributes `category` and `condition`. No reason to reinvent the wheel and do that work over again:
```python
def __init__(self, condition=0):
 super().__init__(category="Clothing", condition=condition)
```
Let's break this down. `super` represents our parent class. We call it with `()` and call the method `__init__`. Then we pass in the necessary arguments the parent class needs to construct the attributes.

Since we know that `Clothing` will always have the category `"Clothing"`, let's not give users the opportunity to change that. Remove the `category` parameter from the child constructor. Now, we as the coder, can always ensure `Clothing` class will always have a category of `"Clothing"`.

Lastly, based on PEP8 styling guide, "Don’t use spaces around the `=` sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter."
```suggestion
    def __init__(self, category="Clothing", condition=0):
```
[source link](https://peps.python.org/pep-0008/#other-recommendations)